### PR TITLE
Adding channel register aliases

### PR DESCRIPTION
**Pull request by truhacevkir**

### CODE REVIEW

File: post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java [hunk 1]
```diff
@@ -84,13 +85,20 @@ public void storeProcessedCollections(String[] collections) {
...
             writeLogsToS3(customer);
         } catch (Exception e) {
             logger.error(PROBLEM_PUSHING_PROCESSED_RECORDS_TO_S3 + e);
         }
     }
 
+    private static void logSuccessfullyProceedRecords(List<Record> records) {
+        logger.info(String.join(StringUtils.EMPTY_STRING, SUCCESSFULLY_PROCEEDED_MESSAGE,
+                                String.valueOf(records.size())));
+    }
```
Comment (truhacevkir) [inline post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java:100]: Adding following log message, cause for now we do not log anything when pushing records to aws only

File: post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java [hunk 2]
```diff
@@ -84,13 +85,20 @@ public void storeProcessedCollections(String[] collections) {
...
+            var records = extractRecords(listOfRecordsCollections);
+            records.forEach(this::storeRecord);
+            logSuccessfullyProceedRecords(records);
             writeLogsToS3(customer);
         } catch (Exception e) {
             logger.error(PROBLEM_PUSHING_PROCESSED_RECORDS_TO_S3 + e);
         }
     }
 
+    private static void logSuccessfullyProceedRecords(List<Record> records) {
```
Comment (joachimjorgensen) [inline post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java:97]:
```suggestion
    private static void logSuccessfullyProceededRecords(List<Record> records) {
```

Comment (joachimjorgensen) [reply post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java:97]: Maybe `logSuccessfullyStoredRecords()` or `logSuccessfullyProcessedRecords()` are better names?

**Review by joachimjorgensen — APPROVED** [review]

File: post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java [hunk 3]
```diff
@@ -87,14 +87,14 @@ public void storeProcessedCollections(String[] collections) {
...
+            logSuccessfullyProcessedRecords(records);
             writeLogsToS3(customer);
         } catch (Exception e) {
             logger.error(PROBLEM_PUSHING_PROCESSED_RECORDS_TO_S3 + e);
         }
     }
 
-    private static void logSuccessfullyProceedRecords(List<Record> records) {
+    private static void logSuccessfullyProcessedRecords(List<Record> records) {
         logger.info(String.join(StringUtils.EMPTY_STRING, SUCCESSFULLY_PROCEEDED_MESSAGE,
```
Comment (joachimjorgensen) [inline post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java:98]: Could consider updating the log message as well, unless it is still correct :)

**Pull request merged**